# Fine_Mapping_Simulation
Using simulated European data from "A new method for multiancestry polygenic prediction improves performance across diverse populations" Zhang et al. Nat. Genet. 2023, SNPs were extracted from chromosome 6 and 15 in Obtain_SimData.R script. The regions extracted are defined in Table S9 from "High-throughput characterization of functional variants highlights heterogeneity and polygenicity underlying lung cancer susceptibility" Long et al. AJHG 2024. Specifically, SNPs were extracted from GWAS locus 11_6p21.33 and 36_15q21.3 in chromosome 6 and 15 respectively. 50 simulated responses were then created in Simulate_Data.R for each GWAS locus. Simulated responses were then analyzed using fine-mapping methods in the Simulation_Study.R script. Lastly, the results from these methods were extracted and plotted using the Simulation_Results.R script.
